frontend/src/app/components/FeaturedExperiment/FeaturedButton.js

```diff
@@ -169,7 +169,7 @@ export default class FeaturedButton extends Component<FeaturedButtonProps, Featu
             <React.Fragment>
               <div className="main-install__spacer"></div>
               <div className="mobile-button-wrap">
-                <MobileTriggerIOSButton doShowMobileAppDialog={this.doShowMobileAppDialog} color={"primary"} />
+                <MobileTriggerIOSButton doShowMobileAppDialog={this.doShowMobileAppDialog} color={"default"} />
```

Comment:
ios is getting different color than android button below

Reply:
I may be misunderstanding the goal of that issue. I'll let @youwenliang or @johngruen review this one

Reply:
Yes I suggested using different colors for the buttons. Look good to me.